README.md

```diff
@@ -2,11 +2,45 @@
 
 A compiler from Python to Mindustry's assembly (logic programming language).
 
-To run the tests, simply use the following command:
+The language **is** Python, so all simple programs you can imagine are possible. Check the [supported features][features]. There also might be some missing, so consider [contributing].
+
+Beyond that, the
+supported "system calls" are documented in the [`pyndustri.pyi`] file.
+
+To learn about the possible compiler errors, refer to the `ERR_` and `ERROR_DESCRIPTIONS`
+constants in the [`constants.py`] file.
+
+## Getting up and running
+
+To install pyndustric as module run:
 
 ```sh
-$ pip install -r dev-requirements.txt
-$ pytest
+$ git clone https://github.com/Lonami/pyndustric.git
+$ pip install -e pyndustric
```

Comment:
By the way I'm pretty sure it's possible to `pip install https://github.com/Lonami/pyndustric/archive/refs/heads/master.zip` directly, which does not even require `git` to be installed.

Also curious, you seem to use `install -e`, which is for "editable" installs.

```
  -e, --editable <path/url>   Install a project in editable mode (i.e. setuptools "develop mode") from a local project
                              path or a VCS url.
```

I don't think users who don't want to develop would want to do this, but maybe you had a reason for it.

Reply:
Reason I did it was because I just copied what I did not thinking much about it. Force of habit
I changed docs now